internal/cloud_connector/message_handlers.go

```diff
@@ -218,6 +218,11 @@ func processDispatchers(logger *logrus.Entry, sourcesRecorder controller.Sources
 	if gotDispatchers == false {
 		logger.Debug("No dispatchers found")
 		return
+	} else {
+		dispatchersMessage, dispatchersMarshalled := json.Marshal(dispatchers)
+		if dispatchersMarshalled == nil {
+			logger.Debugf("Dispatchers found: %s", dispatchersMessage)
+		}
```

Comment:
I think this might be a little easier to read if we take this out of the if/else statement.

Let's add `dispatchers` as a logging field like this:

```
logger = logger.WithFields(logrus.Fields{"dispatchers": dispatchers})
```

How does the log message look if we do not marshal the dispatchers field and just log it as a field?